src/governance.ts

```diff
@@ -74,6 +74,13 @@ export class GovernanceCanister {
     }));
   };
 
+  /**
+   * Returns the list of proposals made for the community to vote on,
+   * paginated and filtered by the request.
+   *
+   * If `certified` is true, the request is fetched as an update call, otherwise
+   * it is fetched using a query call.
+   */
   public listProposals = async (
     request: ListProposalsRequest,
     certified = true
```

Comment:
If true, it uses the one compiled by `compile_certified_did` otherwise `compile_did`? Is that correct?

Reply:
Correct.  I have not kept the certified did file around as I doubt that there is any use for it but I may be surprised!